pkg/framework/test/examples_api_server_custom_pathfinder_test.go

```diff
@@ -0,0 +1,19 @@
+package test_test
+
+import (
+	"fmt"
+
+	. "k8s.io/kubectl/pkg/framework/test"
+)
+
+func ExampleAPIServer_specialPathFinder() {
+	apiServer := &APIServer{
+		PathFinder: myCustomPathFinder,
+	}
+
+	fmt.Println(apiServer)
```

Comment:
Since you don't have a `// Output:` line, this code will not run with tests, not sure this line is useful.